pkg/rc/farm.go

```diff
@@ -133,6 +133,17 @@ func (rcf *Farm) mainLoop(quit <-chan struct{}) {
 
 	rcKeyWatch, rcErr := rcf.rcStore.WatchRCKeysWithLockInfo(subQuit, rcf.rcWatchPauseTime)
 
+	go func(errCh <-chan error) {
+		for {
+			select {
+			case <-quit:
+				return
+			case err := <-rcErr:
```

Comment:
don't think it matters but just noticed this is using the variable from outer scope rather than the arg